subprojects/shipkit/src/main/groovy/org/shipkit/internal/gradle/release/ReleaseNeededPlugin.java

```diff
@@ -19,6 +19,7 @@
  *      <li>SKIP_RELEASE env variable: no release if this environment variable is available</li>
  *      <li>[ci skip-release] commit message: no release if this commit message is available</li>
  *      <li>PR build: no release if the build job is a pull request build</li>
+ *      <li>skipComparePublications task property (ReleaseNeededTask): release if this property is true (although the previous and the current release might be identical)</li>
```

Comment:
It's not entirely true, it doesn't always release (it may still skip release for different reasons).